h/static/scripts/group-forms/components/CreateEditGroupForm.tsx

```diff
@@ -285,7 +287,12 @@ export default function CreateEditGroupForm({
           </>
         )}
 
-        <div className="flex items-center gap-x-4 mt-2">
+        <div className="mt-2 pt-2 border-t border-t-text-grey-6 flex items-center gap-x-4">
+          <span>
+            {/* These are in a child span to avoid a gap between them. */}
```

Comment:
I think this comment is probably not needed.

Reply:
I thought it might be unclear that this un-styled span exists purely to "override" the `gap-x-4` style otherwise. I'm going to err on the side of caution and leave it in.